enforce_codeowners/keras_bot/pull_requests.py

```diff
@@ -29,15 +33,31 @@ def parse_codeowners():
     return map_path_owner
 
 
-def send_message(pull_request, owner, files_changed):
+def send_message(pull_request, owner, files_changed, additions):
     message = """Hello, I'm a bot! I can make mistakes, notify gabrieldemarmiesse if I made one.
 
 I see that you modified the following file{plural}: 
 {files_changed}
 
 The owner of those file is @{owner}
+"""
 
-@{owner} could you please take a look at it whenever 
+    absKeys = list(additions['absolute'].keys())[:10]
+    relKeys = list(additions['relative'].keys())[:10]
+
+    absolute = "The following files should not contain relative imports\n"

```

Comment:
Actually, I'm not certain, but I believe that it's not possible to use relative imports in test files. Meaning that there is quite some code which wouldn't be needed. Could you verify this? Thanks.